plasmapy/formulary/paschen.py

```diff
@@ -0,0 +1,279 @@
+     Wiley-Interscience, New York 1971
+
+     format: paschen_constants dir {"gas":[A,B]}
+     units: A in [Ionisation/(Pa m)] and B in [V/(Pa m)]
+
+
+     `townsend_gamma` is the Second Townsend Ionization coefficient as given by
+     A.Beroual and I. Fonfana, Discharge in Long Air Gap Modeling and Application
+     IOP Publishing Ltd 2016
+     ISBN 978-0-7503-1236-3 (ebook)
+     ISBN 978-0-7503-1237-0 (print)
+
+
+
+     Examples
+     --------
+
+     >>> get_paschen_constants("Ar", "Ni")
+     {'A': 11, 'B': 135, 'gam': 0.058}
+
+     If electrode material is not found a default value of 0.01 is taken
+
+     >>> get_paschen_constants("Ar", "zz")
+     {'A': 11, 'B': 135, 'gam': 0.01}
+
+     If ``gas`` is not found, `None` is returned.
+
+     >>> get_paschen_constants("Zz", "Ni")
+
+
+ """
+
+    # Supported gases
+
+    gases = ["Air", "N2", "H2", "He", "Ne", "Ar", "Kr", "Xe"]
+
+    paschen_constants = {
+        "Air": [11, 274],
+        "N2": [9.0, 257],
+        "H2": [3.8, 104],
+        "N2": [9.0, 257],
+        "He": [2.3, 26],
+        "Ne": [3.0, 75],
+        "Ar": [11, 135],
+        "Kr": [13, 180],
+        "Xe": [20, 263],
+    }
+
+    # Supported electrode materials
+
+    materials = ["Al", "Cu", "Ni", "Pt", "C", "W", "Fe"]
+
+    townsend_gamma = {
+        "Air": {
+            "Al": 0.035,
+            "Cu": 0.025,
+            "Ni": 0.036,
+            "Pt": 0.017,
+            "C": None,
+            "W": None,
+            "Fe": 0.02,
+        },
+        "N2": {
+            "Al": 0.1,
+            "Cu": 0.066,
+            "Ni": 0.077,
+            "Pt": 0.59,
+            "C": None,
+            "W": None,
+            "Fe": 0.059,
+        },
+        "H2": {
+            "Al": 0.095,
+            "Cu": 0.05,
+            "Ni": 0.053,
+            "Pt": 0.02,
+            "C": 0.014,
+            "W": None,
+            "Fe": 0.061,
+        },
+        "He": {
+            "Al": 0.021,
+            "Cu": None,
+            "Ni": 0.015,
+            "Pt": 0.01,
+            "C": None,
+            "W": None,
+            "Fe": 0.015,
+        },
+        "Ne": {
+            "Al": 0.053,
+            "Cu": 0.02,
+            "Ni": 0.031,
+            "Pt": 0.023,
+            "C": None,
+            "W": 0.045,
+            "Fe": 0.022,
+        },
+        "Ar": {
+            "Al": 0.12,
+            "Cu": 0.058,
+            "Ni": 0.058,
+            "Pt": 0.058,
+            "C": None,
+            "W": None,
+            "Fe": 0.058,
+        },
+        "Kr": {
+            "Al": None,
+            "Cu": None,
+            "Ni": None,
+            "Pt": None,
+            "C": None,
+            "W": None,
+            "Fe": None,
+        },
+        "Xe": {
+            "Al": None,
+            "Cu": None,
+            "Ni": None,
+            "Pt": None,
+            "C": None,
+            "W": None,
+            "Fe": None,
+        },
+    }
```

Comment:
If we do go with exceptions or warnings, then we could drop all the `None`s and replace the latter part with a `try/except KeyError` block. Simple to code that up, so I could do it for you, but I'd rather get agreement on how best to proceed here first.